ios/RNPdftron.m

```diff
@@ -52,6 +52,28 @@ - (dispatch_queue_t)methodQueue
     }
 }
 
+RCT_EXPORT_METHOD(getVersion:(RCTPromiseResolveBlock)resolve
+                 rejecter:(RCTPromiseRejectBlock)reject)
+{
+    @try {
+        resolve([@"PDFNet " stringByAppendingFormat:@"%f", [PTPDFNet GetVersion]]);
+    }
+    @catch (NSException *exception) {
+        reject(@"get_failed", @"Failed to get PDFNet version", [self errorFromException:exception]);
+    }
+}
+
+RCT_EXPORT_METHOD(getPlatformVersion:(RCTPromiseResolveBlock)resolve
```

Comment:
Is this something PDFTron would be expected to have? Doesn't seem to be within the scope of our library. Why are we adding this?

Reply:
FL has it (though not in README), so I thought it would be nice to have it in RN as well.

Reply:
OK